demo/config.xml

```diff
@@ -99,6 +99,5 @@
     <plugin name="cordova-plugin-splashscreen" spec="5.0.2" />
     <plugin name="cordova-plugin-ionic-webview" spec="^4.0.0" />
     <plugin name="cordova-plugin-ionic-keyboard" spec="^2.0.5" />
-    <plugin name="cordova-plugin-background-upload" spec="https://github.com/spoonconsulting/cordova-plugin-background-upload.git#afnetworking-spx-demo" />
-    <engine name="android" spec="8.1.0" />
+    <engine name="android" spec="^8.0.0" />
```

Comment:
Pourquoi ?

Reply:
J'avais eu du mal à lancer le projet en local en faite. 
L'installation du cordova-plugin-background-upload ne s'est pas terminé corrèctement. J'ai eu une erreur de type failed to fetch plugin à chaque fois, mais bizzaremment le plugin a bien été déposé dans le répértoire plugins. Du coup, je devrais ajouter manuellement la déclaration du plugin dans le config.xml
Pour l'engine. J'ai dûe faire un remove platform android puis un add platform android sinon le build crachait.